docker/slurm/Dockerfile.base

```diff
@@ -86,5 +88,8 @@ COPY ./docker/slurm/slurmdbd.conf /etc/slurm/slurmdbd.conf
 COPY ./docker/slurm/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 RUN chmod +x /usr/local/bin/docker-entrypoint.sh
 
+COPY ./test.sh /data/test.sh
+COPY ./test2.sh /data/test2.sh
```

Comment:
why should we add this rows? docker-compose doesn't assemble this image now

Reply:
fixed